corehq/toggles.py

```diff
@@ -1011,6 +1011,12 @@ def _commtrackify(domain_name, toggle_is_enabled):
     [NAMESPACE_DOMAIN]
 )
 
+ICDS_CUSTOM_SMS_REPORT = StaticToggle(
+    'icds_custom_sms_report',
+    'ICDS: Generate a custom SMS',
```

Comment:
Can you add more description here?

Reply:
Yeah. Sorry I have left it for the end and it was skipped.
Updated the description in the latest commit.